site/static/docs/5.2/assets/brand/orange-logo-formatted.svg

```diff
@@ -0,0 +1,28 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 283.46 283.46">
+   <defs>
+      <style>
+        #a {
+          fill: #ff7900;
+        }
+
+        #b, #c {
+          fill: #fff;
+        }
+
+        @media (max-width: 49.98px) {
+          #b {
+            display: none;
+          }
+        }
+
+        @media (min-width: 50px) {
+          #c {
+            display: none;
+          }
+        }
+      </style>
+   </defs>
+   <path d="M0 0h283.46v283.46H0z" id="a"/>
+   <path d="M40.51 202.47h202.47v40.5H40.51z" id="c"/>
```

Comment:
Feels weird to have `id="c"` before the `b` one.

Reply:
Prefer to keep the original version created by the designers